suite-native/transactions/src/components/TransactionDetail/TransactionDetailSummary.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box, Card, VStack, Text } from '@suite-native/atoms';
+import { Icon } from '@trezor/icons';
+
+type TransactionDetailSummaryProps = {
+    origin?: string;
+    target?: string;
+    transactionStatus: 'confirmed' | 'pending';
```

Comment:
We should have union type for this already somewhere I guess

Reply:
We actually don't. Or at least I couldn't find it 🥹

Pending transactions get grouped together in a function `groupTransactionsByDate` and you can determine whether tx is pending or not by `isPending` function. But it's never stored in store or assigned to a tx property. 

I guess that's why there is no object property on the transaction itself and no type associated with it.

Reply:
Ok, then leave it as it is, no big deal here it will be typesafe anyway.